src/types/form.ts

```diff
@@ -191,7 +191,7 @@ export interface IFormDocument extends IFormSchema {
   authType: NonNullable<IFormSchema['authType']>
   status: NonNullable<IFormSchema['status']>
   inactiveMessage: NonNullable<IFormSchema['inactiveMessage']>
-  submissionLimit: NonNullable<IFormSchema['submissionLimit']>
+  submissionLimit: Exclude<IFormSchema['submissionLimit'], undefined>
```

Comment:
add comment here saying why nonnullable is not used? (because one of the possible values is null)